contracts/MarketOracle.sol

```diff
@@ -39,6 +39,8 @@ contract MedianOracle is Ownable, IOracle {
     // The number of seconds after which the report is deemed expired.
     uint256 public reportExpirationTimeSec;
 
+    uint256 private constant MAX_REPORT_EXPIRATION_TIME = 10 years;
```

Comment:
Let's put constants either before or after state variables. I think I see both in our codebase...
@nithinkrishna Do you have a preference?

Also, please add a comment about why the value is necessary.

Reply:
It doesn't make a difference to the solidity compiler as storage isn't allocated for constants. 

If we choose to have them at the after, in the case of upgrades when we add new state variables, the new variables go in-between the constants and the old ones. Seems messy.

I prefer having the constants before all the state variables.